pyproject.toml

```
@@ -27,8 +27,11 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "frequenz-api-common >= 0.5.0, < 0.6.0",
-  "googleapis-common-protos >= 1.56.2, < 2",
-  "grpcio >= 1.51.1, < 2",
+  "googleapis-common-protos >= 1.62.0, < 2",
+  "grpcio >= 1.54.2, < 2",
+  "frequenz-channels == 1.0.0b2",
+  "frequenz-client-base@git+https://github.com/frequenz-floss/frequenz-client-base-python@v0.x.x",
```

And this one to the upcoming release.

---

publishing to PyPI failed:  https://github.com/frequenz-floss/frequenz-client-base-python/actions/runs/7301765142/job/19899323856

until that is resolved, could you please use this instead:

```suggestion
  "frequenz-client-base@git+https://github.com/frequenz-floss/frequenz-client-base-python@v0.1.0",
```

---

This has been updated to `frequenz-client-base >= 0.1.0, < 0.2`